test/e2e/specs/wp-launch-spec.js

```diff
@@ -45,32 +32,46 @@ before( async function () {
 describe( `[${ host }] Launch (${ screenSize }) @signup @parallel`, function () {
 	this.timeout( mochaTimeOut );
 
-	describe( 'Launch a free site', function () {
-		const accountName = dataHelper.getNewBlogName();
+	describe( 'Create and launch a free site', function () {
 		const siteName = dataHelper.getNewBlogName();
+		const siteURL = siteName + '.wordpress.com';
 
-		before( 'Create a site as a new user', async function () {
-			await createAndActivateAccount( accountName );
+		step( 'Can log in', async function () {
+			const loginFlow = new LoginFlow( driver );
+			await loginFlow.login();
+		} );
+
+		step( 'Can create one free site as existing user', async function () {
 			await new CreateSiteFlow( driver, siteName ).createFreeSite();
 		} );
 
 		step( 'Can launch a site', async function () {
 			return await new LaunchSiteFlow( driver ).launchFreeSite();
 		} );
 
-		after( 'Delete the newly created account', async function () {
-			return await new DeleteAccountFlow( driver ).deleteAccount( accountName );
+		step( 'Can delete site', async function () {
+			const sidebarComponent = await SidebarComponent.Expect( driver );
+			await sidebarComponent.ensureSidebarMenuVisible();
+			await sidebarComponent.selectSettings();
+			const settingsPage = await SettingsPage.Expect( driver );
+			return await settingsPage.deleteSite( siteURL );
```

Comment:
Since the refactored test do not create a new account to execute the launch flow, for the cleanup process I have instead requested the site(s) be deleted at end.